index.html

```diff
@@ -69,5 +69,184 @@ <h2 class="connect">Let's connect</h2>
+            </p>
+            <ul class="card-tags-container">
+              <li class="card-tag">HTML</li>
+              <li class="card-tag">CSS</li>
+              <li class="card-tag">JavaScript</li>
+            </ul>
+            <div class="card-actions">
+              <a href="#" class="primary-button">See project</a>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="container-about">
+        <div class="me">
+          <h1 class="title">About Myself</h1>
+          <p class="about-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <h2 class="connect">Let's connect</h2>
+          <ul class="social">
+            <li>
+              <a href="twitter.com"
+                ><img src="images/svg/twitter.svg" alt="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="linkedin.com"
+                ><img src="images/svg/Linkedin.svg" alt="LinkedIn"
+              /></a>
+            </li>
+            <li>
+              <a href="https://medium.com"
+                ><img src="images/svg/medium.svg" alt="Medium"
+              /></a>
+            </li>
+            <li>
+              <a href="github.com"
+                ><img src="images/svg/github.svg" alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="angellist.com"
+                ><img src="images/svg/angellist.svg" alt="Angel List"
+              /></a>
+            </li>
+          </ul>
+          <div class="about-actions">
+            <a href="#" class="primary-button">Get my resume</a>
+          </div>
+        </div>
+        <div class="skills">
+          <div class="about-title-container">
+            <h3 class="about-skills">Languages</h3>
+            <img src="/images/svg/arrow.svg" />
+          </div>
+          <ul class="about-languages">
+            <li class="about-item">
+              <img src="/images/languages/JS.png" alt="Javascrip" />JavaScript
+            </li>
+            <li class="about-item">
+              <img src="/images/languages/HTML.png" alt="HTML" />HTML
+            </li>
+            <li class="about-item">
+              <img src="/images/languages/CSS.png" alt="CSS" />CSS
+            </li>
+          </ul>
+          <div class="about-title-container border-b">
+            <h3 class="about-skills">Frameworks</h3>
+            <img src="/images/svg/arrow-right.svg" />
+          </div>
+          <div class="about-title-container border-b">
+            <h3 class="about-skills">skills</h3>
+            <img src="/images/svg/arrow-right.svg" />
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] You have done a great job in implementing the task, however, your social links are not showing, it might have to do with your image path,  Please resolve this.
- A possible approach could be, adding a dot in front of your path directory. `<img src="./images/languages/CSS.png" alt="CSS" />`